frontend/src/views/HomePage/components/FeedArea/FeedArea.styles.js

```diff
@@ -8,6 +8,7 @@ export const FeedArea = styled.main`
   border-radius: 30px;
   background-color: ${({ theme }) => theme.colors.white};
```

Comment:
I think there's no point of storing certain colors in the theme. You can as well use `white`. A color named `white` in the theme will always be white, right?

Reply:
We resigned from using certain colors in the theme.